apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -41,17 +41,6 @@ class SectionTable extends Component {
 
     return (
       <table style={styles.table}>
-        <colgroup>
-          <col width="200"/>
-          <col width="130"/>
-          <col width="98"/>
-          <col width="130"/>
-          <col/>
-          <col/>
-          <col/>
-          <col/>
-          <col/>
-        </colgroup>
```

Comment:
These were copied from the original haml table, and didnt seem to actually be accomplishing anything, so I got rid of them.